assets/stylesheets/join.css

```diff
@@ -1,35 +1,245 @@
-.conn-warn {
-  animation: blink 0.6s cubic-bezier(1, 0, 0, 1) infinite alternate;
-  color:     red;
+body {
+  margin-left: 50px;
+  margin-right: 50px;
 }
 
-@keyframes blink {
-  to { opacity: 0; }
+table {
+  border-collapse: collapse;
+  border-style: hidden;
+  text-align: center;
+  margin: 20px 0 0 0;
+  width: 100%;
 }
 
-.role-select {
-  width: 80%;
+th, td {
```

Comment:
I think nothing in the table (especially not the headers) should be highlightable/copyable.  Check out `user-select: none;`.